src/main/core/manager.rs

```diff
@@ -537,7 +537,7 @@ impl<'a> Manager<'a> {
                 // the manager sets this ID
                 id: host_id,
                 // the manager sets this CPU frequency
-                cpu_frequency: self.raw_frequency_khz,
+                cpu_frequency_khz: self.raw_frequency_khz,
```

Comment:
Should we standardize on Hz instead of Khz?

I regret using Khz (and KiBps in other parts of the code) and it has caused enough confusion over the years that I'd be happy to use base units instead.

Reply:
Yeah, I was debating whether or not to change it. I'll take a look but might push it to a follow-up.

Reply:
Done